test-cases/manager/manager-regression-gce.yaml

```diff
@@ -9,6 +9,8 @@ n_monitor_nodes: 1
 post_behavior_db_nodes: "destroy"
 post_behavior_loader_nodes: "destroy"
 post_behavior_monitor_nodes: "destroy"
+use_preinstalled_scylla: true
+scylla_repo_loader: 'https://s3.amazonaws.com/downloads.scylladb.com/rpm/centos/scylla-4.3.repo'
```

Comment:
isn't the repo only necessary when we have a db node without Scylla installed?

Reply:
The repo is not for the scylla nodes, it's for the loaders, so we can install their prerequisites.